Firestore/Example/Tests/Local/FSTLocalStoreTests.mm

```diff
@@ -367,65 +391,80 @@ - (void)testHandlesPatchWithoutPriorDocument {
+  FSTAssertChanged(@[ FSTTestUnknownDoc("foo/bar", 1) ]);
+  if ([self gcIsEager]) {
+    FSTAssertNotContains(@"foo/bar");
+  } else {
+    FSTAssertContains(FSTTestUnknownDoc("foo/bar", 1));
+  }
 }
 
-- (void)testHandlesPatchMutationThenAckThenDocument {
-  if ([self isTestBaseClass]) return;
-
-  [self writeMutation:FSTTestPatchMutation("foo/bar", @{@"foo" : @"bar"}, {})];
-  FSTAssertRemoved(@[ @"foo/bar" ]);
-  FSTAssertNotContains(@"foo/bar");
-
-  [self acknowledgeMutationWithVersion:1];
-  FSTAssertRemoved(@[ @"foo/bar" ]);
-  FSTAssertNotContains(@"foo/bar");
-
-  FSTQuery *query = FSTTestQuery("foo");
-  TargetId targetID = [self allocateQuery:query];
-
-  [self applyRemoteEvent:FSTTestUpdateRemoteEvent(FSTTestDoc("foo/bar", 1, @{@"it" : @"base"}, NO),
-                                                  @[ @(targetID) ], @[])];
-  FSTAssertChanged(@[ FSTTestDoc("foo/bar", 1, @{@"it" : @"base"}, NO) ]);
-  FSTAssertContains(FSTTestDoc("foo/bar", 1, @{@"it" : @"base"}, NO));
-}
+// TODO(heldwriteacks): Uncomment this test after LocalStore changes
+//- (void)testHandlesPatchMutationThenDocumentThenAck {
+//  if ([self isTestBaseClass]) return;
+//
+//  [self writeMutation:FSTTestPatchMutation("foo/bar", @{@"foo" : @"bar"}, {})];
+//  FSTAssertRemoved(@[ @"foo/bar" ]);
+//  FSTAssertNotContains(@"foo/bar");
+//
+//  FSTQuery *query = FSTTestQuery("foo");
+//  TargetId targetID = [self allocateQuery:query];
+//
+//  [self applyRemoteEvent:FSTTestAddedRemoteEvent(
+//                             FSTTestDoc("foo/bar", 1, @{@"it" : @"base"}, FSTDocumentStateSynced),
+//                             @[ @(targetID) ])];
+//  FSTAssertChanged(@[ FSTTestDoc("foo/bar", 1, @{@"foo" : @"bar", @"it" : @"base"},
+//                                 FSTDocumentStateLocalMutations) ]);
+//  FSTAssertContains(FSTTestDoc("foo/bar", 1, @{@"foo" : @"bar", @"it" : @"base"},
+//                               FSTDocumentStateLocalMutations));
+//
+//  [self acknowledgeMutationWithVersion:2];
+//  // We still haven't seen the remote events for the patch, so the local changes remain, and there
+//  // are no changes
+//  FSTAssertChanged(@[FSTTestDoc("foo/bar", 2, @{@"foo" : @"bar", @"it" : @"base"},
+//      FSTDocumentStateCommittedMutations)]);
+//  FSTAssertContains(FSTTestDoc("foo/bar", 2, @{@"foo" : @"bar", @"it" : @"base"},
+//      FSTDocumentStateCommittedMutations));
+//
+//  [self applyRemoteEvent:FSTTestUpdateRemoteEvent(
+//                             FSTTestDoc("foo/bar", 2, @{@"foo" : @"bar", @"it" : @"base"},
+//                                        FSTDocumentStateSynced),
+//                             @[], @[])];
+//
+//  FSTAssertChanged(
+//      @[ FSTTestDoc("foo/bar", 2, @{@"foo" : @"bar", @"it" : @"base"}, FSTDocumentStateSynced) ]);
+//  FSTAssertContains(
+//      FSTTestDoc("foo/bar", 2, @{@"foo" : @"bar", @"it" : @"base"}, FSTDocumentStateSynced));
+//}
+
+// TODO(heldwriteacks): Uncomment this test after LocalStore changes
```

Comment:
This is all fine, but next time, maybe comment via /* ... */ (or even `#if 0`)? That will cause most lines to not change, making it far easier to review. As is, it's impossible to see what's changed. I've assume that it's mostly just s/NO/FSTDocumentStateSynced/ and the like.